### ISSUE TITLE
[Feature Request] Display stats in the client area

### ISSUE THREAD

**Issue by yagiz-dev** [FOSSBilling/Proxmox#6]

**Comment by Anuril** [comment]
What Kind of stats are you thinking about? 

I'm thinking: 
- Network Traffic In & Out? 
- Uptime stats
- Disk Space Usage
- Backup Space usage (Later)

**Comment by tobsowo** [comment]
- Memory Usage
- Disk Usage
- Traffic In/Out
- CPU Usage
- Load Usage
- Uptime

Others that can be on the dashboard:
- Local and Public IP v4&v6
- OS and kernel version

**Comment by Anuril** [comment]
What frequency do you think? Memory & CPU Load are very dynamic and I don't think they make sense to do "live" because of all the requests that would need to be sent to the API. 
Local & Public IP => Yes, possible
OS & Kernel Version: Only for LXC, because for QEMU it would require information from QEMU Agent. I can look into the possibility to do it if the info is available via the PVE API.

**Comment by tobsowo** [comment]
A graph can work for the memory and CPU load it my not be near real time information but can easily see how much the server is performing.

**Comment by Anuril** [comment]
Yeah, that can work, but for that we'd need to have access to the metrics. SQL isn't the perfect way to store a lot of metrics, so I'm inclined to just build an integration of a grafana dashboards or something like that. In any case, this will be after 0.1.0
